src/elements/content-uploader/UploadInput.tsx

```diff
@@ -42,7 +45,16 @@ const UploadInput = ({
                 data-testid="upload-input"
                 directory={isFolderUpload ? '' : undefined}
                 multiple={isMultiple}
-                onChange={onChange}
+                onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                    const { files } = event.target;
+                    if (onSelection && files) {
+                        const shouldContinue = onSelection(files);
+                        if (!shouldContinue) {
+                            return;
```

Comment:
Is there a way to clear out the file list other than doing `files = ''`?
I know there's clearCache for remote but it'd be helpful to be able to reset the form other than clicking cancel.